### DISCUSSION TITLE
articles/mo-gan-shan-trip

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#130] [Comments]
https://hutusi.com/articles/mo-gan-shan-trip 

 周末去游玩了下口碑颇佳的莫干山，确切的说，并没有进入莫干山风景区，而是在莫干山裸心谷住了两晚。在携程上团购了个套餐，体验了一下这家传说中的网红酒店。不过说实在的，酒店并没有给我惊艳的感觉，反倒是服务让我觉得有些不值这个价格，垒土小屋房间比较小，唯一亮点的是伴着山谷而建，坐在小阳台看书，有点山居的感觉。

**Comment by yansheng836** [comment]
唐之韵

**Comment by marshhj** [comment]
说不定本来就很纯粹，但总有人想着 “过度解读” 而已。

**Comment by hutusi** [comment]
@marshhj 王维的诗确实是很纯粹~

**Comment by hutusi** [comment]
@yansheng836 看了你的网站，自动加载诗句很有意思~~

**Comment by yansheng836** [comment]
> @yansheng836 看了你的网站，自动加载诗句很有意思~~

谢谢，调用了“今日诗词”API
